### PR TITLE
specially mark EIP4844 changes

### PR DESCRIPTION
**Pull request by hwwhww**
We are planning to include additional features (#3182) in the Deneb fork. This PR modifies the previous "[New in Deneb]" comment to "[New in Deneb:EIP4844]", allowing us to differentiate the sources once we integrate these new features.

Usine the notation `EIP4844` rather than `EIP-4844` because it's the form we use for feature spec *constant*, and it's shorter.

### CODE REVIEW

File: specs/deneb/beacon-chain.md [hunk 1]
```diff
@@ -41,16 +41,15 @@
 
 ## Introduction
 
-This upgrade adds blobs to the beacon chain as part of Deneb. This is an extension of the Capella upgrade.
-
-The blob transactions are packed into the execution payload by the EL/builder with their corresponding blobs being independently transmitted and are limited by `MAX_DATA_GAS_PER_BLOCK // DATA_GAS_PER_BLOB`. However the CL limit is independently defined by `MAX_BLOBS_PER_BLOCK`.
```
Comment (hwwhww) [inline specs/deneb/beacon-chain.md:46]: I moved this note to next to `MAX_BLOBS_PER_BLOCK` preset definition.

File: specs/deneb/polynomial-commitments.md [hunk 2]
```diff
@@ -336,7 +336,7 @@ def evaluate_polynomial_in_evaluation_form(polynomial: Polynomial,
 
 ### KZG
 
-KZG core functions. These are also defined in Deneb execution specs.
+KZG core functions. These are also defined in Deneb execution specs for EIP-4844.
```
Comment (djrtwo) [inline specs/deneb/polynomial-commitments.md:339]: The top of this doc mentions "required for Deneb specification". I might be explicit up there that it is for 4844 (within Deneb)

File: specs/deneb/validator.md [hunk 3]
```diff
@@ -98,6 +98,8 @@ All validator responsibilities remain unchanged other than those noted below.
 
 ##### Blob KZG commitments
 
+[New in Deneb:EIP4844]
```
Comment (djrtwo) [inline specs/deneb/validator.md:101]: Seems like we should add more annotations to this doc -- e.g. to "Constructing the `SignedBlobSidecar`s" and the data structures added

**Review by djrtwo — COMMENTED** [review]
nice! seems like a reasonable approach imo

left a couple of nits

**Review by ralexstokes — APPROVED** [review]
I don't know if we _need_ this but I can see how it could be helpful to someone trying to digest the new specs

**Pull request merged**